string type
---
geometry_msgs/Pose position